frontend/app/Lightbox/MyLightbox.jsx

```diff
@@ -37,6 +37,7 @@ class MyLightbox extends CommonSearchView {
         this.bulkSearchDeleteRequested = this.bulkSearchDeleteRequested.bind(this);
         this.userChanged = this.userChanged.bind(this);
         this.redoRestore = this.redoRestore.bind(this);
+        this.displayRedo = this.displayRedo.bind(this);
```

Comment:
you're not accessing `this` in `displayRedo` so you don't need to bind it

Reply:
Removed.